client/src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Grid, makeStyles } from '@material-ui/core';
+import Logo from './Logo';
+import Menu from './Menu';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    background: 'black',
+    [theme.breakpoints.up('xs')]: {
+      height: '8vh',
```

Comment:
This property is leading to some weird display behavior
![Screen Shot 2021-05-19 at 8 26 58 AM](https://user-images.githubusercontent.com/48573314/118812328-fbcdf880-b87b-11eb-92da-735e2cd24f25.png)
I think it would be better to give this constant size in pixels and then below where you set content to 92vh, you can replace with Calc(100vh - $$px)

Reply:
Fixed in the upcoming commit. Could you take a quick look and make sure @dproc96